Sources/WordPressKit/Models/RemoteBlog.swift

```diff
@@ -37,9 +37,12 @@ import Foundation
     /// Features available for the current blog's plan.
     public var planActiveFeatures = [String]()
 
-    /// Indicates whether it's a jetpack site, or not.
+    /// Indicates whether the jetpack connection is active.
```

Comment:
Shouldn't this comment apply to the `jetpackConnection` property?

```swift
/// Indicates whether the jetpack connection is active.
public var jetpack: Bool = false
```

Let's follow the Swift naming conversion and name these something like:

```swift
var isJetpackConnectionActive: Bool { get }
var isJetpackConnected: Bool { get } 
```

```swift
Indicates whether it's a Jetpack connected site.
```

Does this mean that the site is a non-WP.com site?

Reply:
I have updated the API doc to match the properties on the HTTP API https://github.com/wordpress-mobile/WordPressKit-iOS/pull/835/commits/777c6515b1b31dd86f427f7cc3ca41ec920d2add.

> Does this mean that the site is a non-WP.com site?

I believe it's possible that atomic sites have `jetpack_connection == true & jetpack == false`. Just by `jetpack_connection` property alone, we can't tell if a site is a WP.com site.